panel/src/components/View/Buttons/AddUsersButton.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+	<k-button
+		:disabled="!$panel.permissions.users.create"
+		:text="$t('user.create')"
+		icon="add"
+		size="sm"
+		variant="filled"
+		@click="
+			$dialog('users/create', {
+				query: {
+					role: $panel.view.props.role?.id
```

Comment:
After seeing a few of such dependency on view props, I wonder if we might need a system that a view can provide configuration for potential buttons and then the buttons can specifically check for those.